src/storage/mvcc/mod.rs

```diff
@@ -36,28 +36,41 @@ quick_error! {
             cause(err)
             description(err.description())
         }
-        KeyIsLocked {key: Vec<u8>, primary: Vec<u8>, ts: u64, ttl: u64} {
+        KeyIsLocked { key: Vec<u8>, primary: Vec<u8>, ts: u64, ttl: u64 } {
             description("key is locked (backoff or cleanup)")
-            display("key is locked (backoff or cleanup) {}-{}@{} ttl {}",
+            display("key is locked (backoff or cleanup) {:?}-{:?}@{} ttl {}",
                         escape(key),
                         escape(primary),
                         ts,
                         ttl)
         }
-        BadFormatLock {description("bad format lock data")}
-        BadFormatWrite {description("bad format write data")}
-        Committed {commit_ts: u64} {
+        BadFormatLock { description("bad format lock data") }
+        BadFormatWrite { description("bad format write data") }
+        Committed { commit_ts: u64 } {
             description("txn already committed")
             display("txn already committed @{}", commit_ts)
         }
-        TxnLockNotFound {start_ts: u64, commit_ts: u64, key: Vec<u8> } {
+        Rollbacked { start_ts: u64, key: Vec<u8> } {
```

Comment:
I think better to use `PessimisticLockRollbacked`, because prewrite also can be rollbacked.